templates/inventory-container-infra.j2

```diff
@@ -1,6 +1,12 @@
 [openstack]
 localhost   ansible_connection='local'    ansible_python_interpreter='/usr/bin/env python'
 
+{% if os_container_infra_environment_group is not none %}
```

Comment:
You'll have to show me how do use this. The purpose is not entirely clear to me yet.

Reply:
If you set it to the name of an ansible group, all hosts will be added to that group. There will be more context in a PR to p3-appliances